examples/controls/index.html

```diff
@@ -0,0 +1,21 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset='UTF-8' />
+    <title>react-map-gl Controls Example</title>
+    <style>
+  body {
```

Comment:
Wonder if it is better to do these here in CSS or programatically. Maybe create a css file to make it a little easier to find?